app-connector-client/server/config.js

```diff
@@ -36,6 +38,19 @@ function configValidation(configJson) {
                 if (!event.specification.match(/^.+\.(json|yaml|yml)$/)) {
                     error_message += "\nevent '" + event.name + "': specification '" + event.specification + "' does not match pattern '^.+\\.(json|yaml|yml)$'";
                 }
+                else {
+                    var specInJson
+                    if (event.specification.endsWith(".json")) {
+                        specInJson = JSON.parse(fs.readFileSync(event.specification))
+                    } else {
+                        specInJson = yaml.safeLoad(fs.readFileSync(event.specification, 'utf8'));
+                    }
+                    check_api.check_api(specInJson, {}, function (err, options) {
+                        if (err) {
+                            error_message += "\nevent number " + i + ": Schema validation Error \n" + JSON.stringify(err)
```

Comment:
please use event name, not number.

event '" + event.name + "'

Reply:
done